src/Listener/InMemorySolutionListener.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpSchool\PhpWorkshop\Listener;
+
+use PhpSchool\PhpWorkshop\Event\ExerciseRunnerEvent;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Move solution to temp and replace program arg
+ */
+class InMemorySolutionListener
```

Comment:
in fact I can't remember what this is for at all, maybe old WIP stuff, or maybe it's super important

Reply:
OOH I remember ... this is looking at putting users solution into temp... which is not feasible so needs to be removed... 

Essentially a users solution could be placed anywhere on their machine, so copying the whole dir is out of the question, we could end up moving the whole home dir into temp 😱

Reply:
Oh yeah that would be ridiculous 😂